src/core/components/parameter-row.jsx

```diff
@@ -1,9 +1,12 @@
 import React, { Component } from "react"
-import { Map } from "immutable"
+import { Map, List } from "immutable"
 import PropTypes from "prop-types"
 import ImPropTypes from "react-immutable-proptypes"
 import win from "core/window"
-import { getExtensions, getCommonExtensions, numberToString } from "core/utils"
+import { getExtensions, getCommonExtensions, numberToString, stringify } from "core/utils"
+
+import ExamplesSelectValueRetainer from "./examples-select-value-retainer"
```

Comment:
**Change request**:  expose and consume new Examples-related components via the plugin interface